### PR TITLE
[Hotfix][Oss File Connector] fix oss connector can not run bug

### PR DESCRIPTION
**Pull request by EricJoy2048**
### Purpose of this pull request

1. Improve jar dependency of oss file connector.
2. Improve oss connector document.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Have no change the code, so no need test.

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  3. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  4. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: docs/en/connector-v2/sink/OssFile.md [hunk 1]
```diff
@@ -22,61 +33,114 @@ By default, we use 2PC commit to ensure `exactly-once`
...
+| Map                  | Map                   |
 
-## Supported DataSource Info
+## Options
 
-In order to use the OssFile connector, the following dependencies are required.
-They can be downloaded via install-plugin.sh or from the Maven central repository.
+|               name               |  type   | required |               default value                |                                                      remarks                                                      |
+|----------------------------------|---------|----------|--------------------------------------------|-------------------------------------------------------------------------------------------------------------------|
+| path                             | string  | yes      | -                                          |                                                                                                                   |
```
Comment (zhilinli123) [inline docs/en/connector-v2/sink/OssFile.md]: Add an extra Description?

Comment (EricJoy2048) [reply docs/en/connector-v2/sink/OssFile.md]: > Add an extra Description?

Done.

File: seatunnel-dist/pom.xml [hunk 2]
```diff
@@ -723,15 +748,6 @@
...
                 </dependency>
-
-                <!-- hadoop jar -->
-                <dependency>
-                    <groupId>org.apache.seatunnel</groupId>
-                    <artifactId>seatunnel-hadoop3-3.1.4-uber</artifactId>
-                    <version>${project.version}</version>
-                    <classifier>optional</classifier>
-                    <scope>provided</scope>
-                </dependency>
```
Comment (hailin0) [inline seatunnel-dist/pom.xml]: This jar is currently packaged into the release binary

Comment (TyrantLucifer) [reply seatunnel-dist/pom.xml]: Agree with @hailin0

Comment (EricJoy2048) [reply seatunnel-dist/pom.xml]: Ok, I will add it back. But we need add hadoop license to `LICENSE` file. I created an issue for it and need done before 2.3.4 version release. 
https://github.com/apache/seatunnel/issues/6036

Comment (EricJoy2048) [reply seatunnel-dist/pom.xml]: Done.

**Review by Hisoka-X — APPROVED** [review]

**Review by hailin0 — APPROVED** [review]

**Pull request merged**